environments/production/postgresql.yml

```diff
@@ -54,12 +55,14 @@ dbs:
     pgbouncer_endpoint: pgucr_nlb
     pgbouncer_hosts:
       - pgbouncer7
+      - pgbouncer6
     query_stats: True
   synclogs:
     host: rds_pgsynclog0
     pgbouncer_endpoint: pgsynclogs_nlb
     pgbouncer_hosts:
       - pgbouncer6
+      - pgbouncer7
   auditcare:
     host: rds_pgauditcare0
     pgbouncer_endpoint: pgmain_nlb
```

Comment:
I think you want to add `pgbouncer6` here as well, since the auditcare db also uses the pgmain_nlb (which is fine)